octobot_trading/personal_data/positions/position.py

```diff
@@ -218,6 +221,9 @@ async def update(self, update_size=None, mark_price=None, update_margin=None):
             else:
                 await self.close()
         except errors.LiquidationPriceReached:
+            size_update = self.get_quantity_to_close()
+            self.update_average_exit_price(size_update, self.mark_price)
```

Comment:
I think we should catch `InvalidOperation` and probably `DivisionByZero` here. We should put this in a method (linked to other comments)